packages/babel-plugin-transform-destructuring/src/util.ts

```diff
@@ -193,7 +248,7 @@ export class DestructuringTransformer {
     ) {
       return node;
     } else {
-      return this.scope.toArray(node, count, this.arrayLikeIsIterable);
+      return toArray(this.scope, node, count, this.arrayLikeIsIterable);
```

Comment:
Can we maybe inline the `toArray` function here, given that this method is very short and already called `toArray`?